production.ini.in

```diff
@@ -38,6 +38,7 @@ sqlalchemy.lubis.url = postgresql://${vars:dbhost}:${vars:dbport}/lubis
 mako.directories = chsdi:templates
 
 data_path = /var/local/cartoweb/
+zadara_downloads = /var/local/cartoweb/downloads/
```

Comment:
usually, if the variable does not change according to the env, we put it here directly.
See also `address_search_referers`